packages/bpk-component-checkbox/src/BpkCheckbox.js

```diff
@@ -49,6 +52,10 @@ const BpkCheckbox = (props) => {
         disabled={disabled}
         {...rest}
       />
+      <Tick
+        className={getClassName('bpk-checkbox__icon')}
+        style={{ width: bpkSpacingSm, height: bpkSpacingSm }}
```

Comment:
Why set width/height via inline style vs scss?

Reply:
Because the component itself sets it via style and that overrides any values set from scss

Reply:
Cool ok - just acknowledging the hackety hack,